mne/report.py

```diff
@@ -815,12 +821,17 @@ def add_images_to_section(self, fnames, captions, scale=None,
         section : str
             Name of the section. If section already exists, the images
             will be appended to the end of the section.
+        textbox : None | str | list of str
+            A string of text or a list of strings of text to be appended after
+            the image.
         """
         # Note: using scipy.misc is equivalent because scipy internally
         # imports PIL anyway. It's not possible to redirect image output
         # to binary string using scipy.misc.
         from PIL import Image
         fnames, captions = self._validate_input(fnames, captions, section)
+        if isinstance(textbox, list):
+            textbox = '<br>'.join(textbox)
```

Comment:
English only ;) jk. currently it seems to be a problem with `u'` because it must combine with other unicode strings. I was able to render an accented letter but it was the one I had intended.